docs/tasks/issuers/index.rst

```diff
@@ -0,0 +1,74 @@
+==================
+Setting up Issuers
+==================
+
+Before you can begin issuing certificates, you must configure at least one
+Issuer or ClusterIssuer resource in your cluster.
+
+These represent a certificate authority from which signed x509 certificates can
+be obtained, such as Let's Encrypt, or your own signing key pair stored in a
+Kubernetes Secret resource. They are referenced by Certificate resources in
+order to request certificates from them.
+
+An :doc:`Issuer </reference/issuers>` is scoped to a single namespace, and can
+only fulfill :doc:`Certificate </reference/certificates>` resources within its
+own namespace. This is useful in a multi-tenant environment where multiple
+teams or independent parties operate within a single cluster.
+
+On the other hand, a :doc:`ClusterIssuer </reference/clusterissuers>` is a
+cluster wide version of an :doc:`Issuer </reference/issuers>`. It is able to be
+referenced by :doc:`Certificate </reference/certificates>` resources in any
+namespace.
+
+Users often create ``letsencrypt-staging`` and ``letsencrypt-prod``
+:doc:`ClusterIssuers </reference/clusterissuers>` if they operate a
+single-tenant environment and want to expose a cluster-wide mechanism for
+obtaining TLS certificates from `Let's Encrypt`_.
+
+Supported issuer types
+======================
+
+cert-manager supports a number of different issuer backends, each with their
+own different types of configuration.
+
+Please follow one of the below linked guides for to learn how to set up
+the issuer types you require:
+
+* :doc:`CA <./setup-ca>` - issue certificates signed by a X509 signing keypair,
+  stored in a Secret in the Kubernetes API server.
+* :doc:`Self signed <./setup-selfsigned>` -
+* :doc:`ACME <./setup-acme>` -
+* :doc:`Vault <./setup-vault>`-
+
+Additional information
+======================
+
+There are a few key things to know about Issuers, but for full information
+you can refer to the :doc:`Issuer reference docs </reference/issuers>`.
+
+Difference between Issuers and ClusterIssuers
```

Comment:
A lot of this is mentioned above in `Setting up Issuers`, it might be worth consolidating what is said there into this section, then moving it up to before `Supported issuer types`.